geographic_msgs/GeoPoint coordinates
---
bool is_updated